src/schemas/compare.py

```
@@ -1,4 +1,24 @@
+│   └── summary: 해당 국가 법률 요약
+└── compare_summary (CompareAnalysis): 비교 분석 결과
+    ├── common: 공통점
+    └── diff: 차이점
+
+[왜 중첩 구조인가?]
+프론트엔드에서 국가별 요약과 비교 분석을 구분하여 각각 다른 UI 영역에
+표시할 수 있도록 데이터를 구조화했습니다.
+"""
+
 from pydantic import BaseModel
 from typing import List
```

Ruff (0.15.2)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)